lib/cbus_elixir_web/templates/page/index.html.eex

```diff
@@ -13,7 +13,7 @@
         
         <h2 class="title myriad">This Month's Speakers</h2>
         <ul class="list">
-            <%= Enum.map(meeting_speakers().speakers, fn(s) -> %>
+            <%= Enum.map(@next_meeting.speakers, fn(s) -> %>
```

Comment:
nil handling

Reply:
Switched to using `one!` like we were before, so this will always have a record.